composables/collectionActivity/useCollectionActivity.ts

```diff
@@ -1,27 +1,35 @@
 import { Flippers, InteractionWithNFT, Offer, Owners } from './types'
 import { getFlippers, getOwners } from './helpers'
 import { nameWithIndex } from '@/utils/nft'
+import collectionActivityEvents from '@/queries/subsquid/general/collectionActivityEvents.graphql'
 
-export const useCollectionActivity = ({ collectionId }) => {
-  const { urlPrefix } = usePrefix()
+export const useCollectionActivity = ({
+  collectionId,
+}: { collectionId?: string } = {}) => {
+  const { client } = usePrefix()
   const events = ref<InteractionWithNFT[]>([])
   const owners = ref<Owners>()
   const flippers = ref<Flippers>()
   const offers = ref<Offer[]>([])
+  const { drop } = storeToRefs(useDropStore())
 
-  const queryPrefixMap = {
-    ksm: 'chain-ksm',
-  }
-
-  const queryPrefix = queryPrefixMap[urlPrefix.value] || 'subsquid'
-
-  const { data } = useGraphql({
-    queryPrefix,
-    queryName: 'collectionActivityEvents',
-    variables: {
-      id: collectionId,
```

Comment:
The `collectionId` was ignored here. It would cause the activity section dose not work on collection details page.

Reply:
check `components/collection/activity/Activity.vue` line 52

Reply:
@Jarsen136  DM'd you in tg